src/pages/movie-detail/MovieDetail.css

```diff
@@ -0,0 +1,69 @@
+.detail-section {
+  border: black 1px solid;
+  position: relative;
+  width: 100vw;
+  height: 100vh;
+  background-size: cover;
+}
+
+.detail-content {

```

Comment:
Have another look at how your movie details page look in small mobile. It looks like it has to do with the position: absolute; of this element:

<img width="515" alt="Skärmavbild 2024-04-10 kl  17 35 54" src="https://github.com/Technigo/project-movies-vite/assets/51529201/f6f91ed8-438e-45b3-9ca1-a8c1916404da">